pkgs/by-name/mo/moneydance/package.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchzip, makeWrapper, openjdk21, openjfx21, }:
+let jdk = openjdk21.override { enableJavaFX = true; };
+in stdenv.mkDerivation (finalAttrs: {
+  pname = "moneydance";
+  version = "2023.3_5064";
+
+  src = fetchzip {
+    url = "https://infinitekind.com/stabledl/${finalAttrs.version}/moneydance-linux.tar.gz";
+    hash = "sha256-jHr1V/gV1seenw2Q0/G405lTiabEYEsOS8p/XyByrtM=";
+  };
+
+  passthru = { inherit jdk; };
```

Comment:
`passthru` is usally defined right above the `meta` block.

Reply:
Oh, that makes sense. Done!